tasks/main.yml

```diff
@@ -16,8 +16,15 @@
     - node_exporter_configure
     - node_exporter_run
 
+- name: Gather currently installed node_exporter version (if any)
+  command: "/usr/local/bin/node_exporter --version &>/dev/stdout|grep 'version {{ node_exporter_version }}'|awk '{print $3}'"
```

Comment:
`command` module doesn't allow usage of pipes (`|`), this is what `shell` module is for. However don't try doing this with `shell` as this won't be accepted in this repository.

Reply:
It might be better to register output of `/usr/local/bin/node_exporter --version` and then do extraction while doing comparison. Sth similar to:

```python
__node_exporter_current_version.stdout_lines[0].split(" ")[2] != node_exporter_version
```

Reply:
@paulfantom thanks! Do you have any suggestion how I can use "tox" without installing all of it on my system? I am trying to avoid having all kinds of dependencies, I would usually use docker to do that, but I am also a python noob. ;)

Reply:
Same for me, but testing this won't be easy inside docker as we are launching docker containers. That's why tox is great, you only need python and python tox installed not even ansible is needed. Then when you run `tox` it will contain everything inside virtual environment (no polluting your host) and start molecule, which will spawn docker containers. After tests are completed (success or fail) containers are deleted, but I don't remember what happens to python virtual environments, they probably are staying in `.tox` directory

Reply:
Docker in docker should work though. I am giving tox directly a try though. Will try to fix this up. Let me know when you have a chance to review my last commit, just wondering if this is what you're looking for.

Reply:
That's exactly what I had in mind :+1:

Reply:
OK, great. I am pushing again — at least I got past `ansible-lint`. But then later on `tox` fails or hangs at: 

```
objc[81068]: +[__NSPlaceholderDate initialize] may have been in progress in another thread when fork() was called.
objc[81068]: +[__NSPlaceholderDate initialize] may have been in progress in another thread when fork() was called. We cannot safely call it or ignore it in the fork() child process. Crashing instead. Set a breakpoint on objc_initializeAfterForkError to debug.
```

I've seen that before, but I don't have the time to do get to the bottom of this now. Hope Travis builds, then I can squash/clean-up if it's to your liking.

Reply:
So you are working on a Mac :D

Follow this: https://github.com/ansible/ansible/issues/31869